.github/workflows/publish-image.yml

```diff
@@ -0,0 +1,55 @@
+name: Docker image
```

Comment:
Please remove merge commit from this PR, makes it a bit difficult to review. 

I'm planning to do rebase merge of `dev` branch into `main` during next release (which I'd target after merging this feature). Unfortunately it'll also require force pull & rebase for anyone developing on it, but I think that won't be a big issue 🤔 Would like to keep the history linear rather than having lots of merging between dev and main. Didn't consider that when I accepted the previous PR to main branch 🤷‍♂️

Reply:
Makes sense, will drop it. Only had it there because it makes it easier to test.